shell/platform/linux/fl_value.cc

```diff
@@ -92,6 +92,120 @@ static ssize_t fl_value_lookup_index(FlValue* self, FlValue* key) {
   return -1;
 }
 
+static void int_to_string(GString* buffer, int64_t value) {
```

Comment:
Style nit for these: C++ style guide requires in/out params to be after input params.